Parse/src/main/java/com/parse/ParseFileUtils.java

```diff
@@ -519,6 +519,11 @@ public static String readFileToString(File file, String encoding) throws IOExcep
     return readFileToString(file, Charset.forName(encoding));
   }
 
+  public static void writeStringToFile(File file, String string)
+      throws IOException {
+    writeByteArrayToFile(file, string.getBytes());
+  }
+
```

Comment:
I don't think we should add this since it was `@Deprecated` in `org.apache.commons.io.FileUtils`